### ISSUE TITLE
Glympse?

### ISSUE THREAD

**Issue by StevenBlack** [lightswitch05/hosts#1]
Hi Daniel @lightswitch05 

The folks at Glympse are asking why their service and app is blocked?  Can you look into this?

This is a full-on block  😄   I use Glympse occasionally....

```
glympse.com
api.glympse.com
motd.api.glympse.com
blog.glympse.com
business.glympse.com
content.glympse.com
dev.glympse.com
api.dev.glympse.com
www.api.dev.glympse.com
developer.glympse.com
enterprise.glympse.com
hackathon.glympse.com
api.ipv6.glympse.com
product.glympse.com
itt.proxy.glympse.com
kibana.n0.production.proxy.glympse.com
itt.s0.production.proxy.glympse.com
kibana.s0.production.proxy.glympse.com
kibana.s1.production.proxy.glympse.com
kibana.n0.sandbox.proxy.glympse.com
kibana.s0.sandbox.proxy.glympse.com
kibana.s1.sandbox.proxy.glympse.com
remote.glympse.com
sandbox.glympse.com
api.sandbox.glympse.com
motd.api.sandbox.glympse.com
services.glympse.com
snapshot.glympse.com
upload.glympse.com
east.vpn.glympse.com
west.vpn.glympse.com
www.glympse.com
```

**Comment by lightswitch05** [comment]
Yeah I added the root domain and it discovered the rest. Is this not a analytics and tracking company?

**Comment by StevenBlack** [comment]
Glympse, as I use it, is an app you use to voluntarily share your location with others.  

I've used it on trips where a bunch of cars convoy to a ski holiday, for example.  Also on holiday when the family spreads out over the city so we know where the others are.  It helps groups of people stay spacially organized.

**Comment by lightswitch05** [comment]
From the privacy policy:

> We may partner with a third party to either display advertising on our Website or within our apps or to manage advertising on other sites or within other apps. Our third-party partner may use technologies such as cookies to gather information about your activities on this site or within our app and on other sites or within other apps in order to provide you advertising based upon your browsing activities and interests.

Since I didn't remember what website/app I saw the domain come across I'll just remove them all, but maybe I can do a more targeted block later for just ads and analytics

**Comment by lightswitch05** [comment]
Closed by #2
